apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -0,0 +1,487 @@
+import * as React from 'react';
+import {
+  usePopper,
+  createArrowStyles,
+  PositioningProps,
+  PopperVirtualElement,
+  PopperRefHandle,
+} from '@fluentui/react-positioning';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { useMergedRefs } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { storiesOf } from '@storybook/react';
+import { useFluent } from '@fluentui/react-shared-contexts';
+
+const useStyles = makeStyles({
+  wrapper: {
+    display: 'flex',
+    ...shorthands.gap('5px'),
+    backgroundColor: tokens.colorNeutralBackground1,
+
+    '& .target': {
+      color: tokens.colorNeutralForeground1,
+      backgroundColor: tokens.colorNeutralBackground1,
+      ...shorthands.border('2px', 'dashed', 'green'),
+      width: '400px',
+      height: '200px',
+    },
+  },
+
+  gridWrapper: {
+    width: '400px',
+    height: '400px',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto',
+    columnGap: '20px',
+    rowGap: '50px',
+  },
+
+  boundary: {
+    ...shorthands.border('2px', 'dashed', 'red'),
+  },
+
+  box: {
+    ...shorthands.padding('15px'),
+    ...shorthands.border('1px', 'solid', 'blue'),
+    backgroundColor: 'white',
+  },
+
+  arrow: {
+    ...createArrowStyles({ arrowHeight: 5 }),
+    backgroundColor: 'red',
+  },
+
+  seeThrough: {
+    opacity: 0.6,
+  },
+});
+
+const positions = [
+  ['above', 'start'],
+  ['above', 'center'],
+  ['above', 'end'],
+  ['below', 'start'],
+  ['below', 'center'],
+  ['below', 'end'],
+  ['before', 'top'],
+  ['before', 'center'],
+  ['before', 'bottom'],
+  ['after', 'top'],
+  ['after', 'center'],
+  ['after', 'bottom'],
+] as const;
+
+const Box = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>((props, ref) => {
+  const styles = useStyles();
+  return (
+    <div {...props} className={mergeClasses(styles.box, props.className)} ref={ref}>
+      {props.children}
+    </div>
+  );
+});
+
+const PositionAndAlignProps: React.FC<{ positionFixed?: boolean }> = ({ positionFixed }) => {
+  const styles = useStyles();
+  const positionedRefs = positions.reduce<ReturnType<typeof usePopper>[]>((acc, cur) => {
+    const popperOptions: PositioningProps = { position: cur[0], align: cur[1] };
+    // positionFixed is not public yet
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    popperOptions.positionFixed = positionFixed;
```

Comment:
`positionFixed` and `unstable_disableTether` are not exported publicly, if there is a better way of doing this, I'm all ears